src/index.html

```diff
@@ -11,7 +11,142 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"    method="post">
```

Comment:
Only one space between attributes
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```